test/integration/02-user-wallet/02-index.spec.ts

```diff
@@ -117,7 +117,7 @@ describe("UserWallet", () => {
 
   describe("getStringCsv", () => {
     const csvHeader =
-      "voided,approved,_id,accounts,credit,debit,_journal,book,unix,date,datetime,currency,username,type,hash,txid,fee,feeUsd,sats,usd,memo,memoPayer,meta,pending"
+      "voided,approved,_id,accounts,credit,debit,_journal,book,unix,date,datetime,currency,username,type,hash,txid,fee,feeUsd,sats,usd,memo,memoFromPayer,meta,pending"
```

Comment:
please check with @nicolasburtey if this breaks some integration (dashboard)

Reply:
This was a question I had as well when making the changes. I think it's probably safer to change the two string values of this back to the original for now.